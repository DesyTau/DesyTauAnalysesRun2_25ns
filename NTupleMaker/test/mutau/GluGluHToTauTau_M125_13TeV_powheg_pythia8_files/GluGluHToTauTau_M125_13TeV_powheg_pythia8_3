/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_127.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_128.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_129.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_130.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_131.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_132.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_133.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_134.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_135.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_136.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_137.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_138.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_139.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_140.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_141.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_142.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_143.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_144.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_145.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_146.root
