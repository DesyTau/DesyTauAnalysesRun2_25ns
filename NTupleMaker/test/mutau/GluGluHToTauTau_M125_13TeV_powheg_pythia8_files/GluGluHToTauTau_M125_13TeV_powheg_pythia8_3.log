000 (2509988.000.000) 01/17 11:47:05 Job submitted from host: <131.169.223.40:9618?addrs=131.169.223.40-9618+[2001-638-700-10df--1-28]-9618&noUDP&sock=7812_f5f0_32>
...
022 (2509988.000.000) 01/17 11:48:54 Job disconnected, attempting to reconnect
    Socket between submit and execute hosts closed unexpectedly
    Trying to reconnect to slot1@bird149.desy.de <131.169.84.110:38297?addrs=131.169.84.110-38297>
...
024 (2509988.000.000) 01/17 11:48:54 Job reconnection failed
    Job not found at execution machine
    Can not reconnect to slot1@bird149.desy.de, rescheduling job
...
001 (2509988.000.000) 01/17 11:49:36 Job executing on host: <131.169.163.127:34187?addrs=131.169.163.127-34187>
...
006 (2509988.000.000) 01/17 11:49:46 Image size of job updated: 176372
	119  -  MemoryUsage of job (MB)
	121784  -  ResidentSetSize of job (KB)
...
006 (2509988.000.000) 01/17 11:49:51 Image size of job updated: 203236
	119  -  MemoryUsage of job (MB)
	121816  -  ResidentSetSize of job (KB)
...
005 (2509988.000.000) 01/17 11:49:51 Job terminated.
	(1) Normal termination (return value 255)
		Usr 0 00:00:12, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:12, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	0  -  Run Bytes Sent By Job
	0  -  Run Bytes Received By Job
	0  -  Total Bytes Sent By Job
	0  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     0.61        1         1 
	   Disk (KB)            :     1     3072000   3928266 
	   Memory (MB)          :   119        2048      2048 
...
