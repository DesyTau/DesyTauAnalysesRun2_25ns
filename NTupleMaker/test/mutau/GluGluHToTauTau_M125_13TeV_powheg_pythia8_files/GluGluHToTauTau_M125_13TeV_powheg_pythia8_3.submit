+RequestRuntime=10000

RequestMemory = 2000

executable = GluGluHToTauTau_M125_13TeV_powheg_pythia8_3.sh

transfer_executable = True
universe            = vanilla
getenv              = True
Requirements        = OpSysAndVer == "SL6"

output              = GluGluHToTauTau_M125_13TeV_powheg_pythia8_3.out
error               = GluGluHToTauTau_M125_13TeV_powheg_pythia8_3.error
log                 = GluGluHToTauTau_M125_13TeV_powheg_pythia8_3.log

queue

