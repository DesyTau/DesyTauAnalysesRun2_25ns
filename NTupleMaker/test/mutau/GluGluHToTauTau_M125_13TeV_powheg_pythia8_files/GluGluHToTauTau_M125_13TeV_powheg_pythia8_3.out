/nfs/dust/cms/user/cardinia/HtoTauTau/HiggsCP/CMSSW_9_4_9/src/DesyTauAnalyses/NTupleMaker/test/mutau/GluGluHToTauTau_M125_13TeV_powheg_pythia8_files
Number of single lepton trigger legs = 2
Number of X trigger legs (lep leg)   = 2
Number of X trigger legs (tau leg)   = 2
Number of ditau trigger legs         = 0
dxyLeptonCut 0.045
dzLeptonCut 0.2
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_127.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_128.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_129.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_130.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_131.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_132.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_133.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_134.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_135.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_136.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_137.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_138.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_139.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_140.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_141.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_142.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_143.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_144.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_145.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_146.root
Histogram GluGluHToTauTau_M125_13TeV_powheg_pythia8_pileup is not present in pileup file
